src/storage/keyvalue/WrappedExpiringStorage.ts

```diff
@@ -88,4 +109,11 @@ export class WrappedExpiringStorage<TKey, TValue> implements ExpiringStorage<TKe
     }
     return result;
   }
+
+  /**
+   * Stops the continuous cleanup timer.
+   */
+  public quit(): void {
```

Comment:
```suggestion
  public finalize(): void {
```
seems to be more common

Reply:
Shall we also have a `Finalizable` interface to track all such classes?

Reply:
Also thinking that we _could_ have a big `Finalizer` object in config that takes all `Finalizable` instances as an argument, and we call that one object when the server stops.